### ISSUE TITLE
Package name collision

### ISSUE THREAD

**Issue by morganwillcock** [ryrun/kixtart-mode#2]
Hello,

I am working on a KiXtart package for Emacs which has used the same package name of "kixtart-mode":
https://github.com/morganwillcock/kixtart-mode

It is approaching the point where I consider it finished and I would like to try and submit it to GNU ELPA. Would you be willing to effectively release the name of "kixtart-mode" so that I can try to submit my package as a direct replacement?

Thanks,
Morgan

**Comment by ryrun** [comment]
Sure, but how can i release the the name?

**Comment by morganwillcock** [comment]
If you were happy to let me use the package name, I would speak to some at MELPA to ask the best way to proceed, with the aim being to remove the existing package at the same time the replacement package is made available.

**Comment by ryrun** [comment]
OK, yes, you can use the name. I dont use this package anymore.

**Comment by morganwillcock** [comment]
Thanks! I'll try to speak to someone about the change and direct them to look here as an indication that you are happy with the change.

**Comment by ryrun** [comment]
> Thanks! I'll try to speak to someone about the change and direct them to look here as an indication that you are happy with the change.

Any news about the change?

**Comment by morganwillcock** [comment]
Not yet. I needed to go through the FSF copyright assignment process before attempting to submit the package - that part was completed today. So I am now at the stage where I can make the submission and request the switch...

**Comment by morganwillcock** [comment]
Unfortunately the package cannot be added to GNU ELPA because of the nature of what it does, so that has foiled my plan. I do not use MELPA so I'm not really motivated to maintain a package on it, and if I do not submit the package to any public repository then it doesn't make sense for me to take over over the package name.

In summary, I do not think I am going to submit the package anywhere else. If you want to the leave your existing package up, that is fine with me. Apologies for the noise.

**Comment by ryrun** [comment]
No problem. Maybe start with a new name? Like the "php-mode-improved" one?

**Comment by morganwillcock** [comment]
Maybe, I'll have a think about it.
For the moment I'm just leaving it as "manual installation only".
Thanks for your time.

**Comment by morganwillcock** [comment]
Hi,

Sorry to bother you again with the same topic, but I've continued to work on the package over the last few years. Given that OS packages are being automatically generated from MELPA I would like to take over the kixtart-mode MELPA entry and switch my package in for yours.

Would you be happy if I submit a pull request to MELPA to replace the repository reference?

Thanks,
Morgan

**Comment by ryrun** [comment]
Hi,

That’s fine with me. That means you would submit a change to the MELPA GitHub repository for the KiXtart recipe, which would then point to your fork, right? Shouldn’t it be publicly accessible?

Best regards

**Comment by morganwillcock** [comment]
> That’s fine with me. That means you would submit a change to the MELPA GitHub repository for the KiXtart recipe, which would then point to your fork, right?

Yes, although the repository is not a fork. I will submit the change now.

> Shouldn’t it be publicly accessible?

I moved my personal projects to Sourcehut. Here is the link if you would like to take a look: https://git.sr.ht/~mew/kixtart-mode
